src/dissect/dissect.c

```diff
@@ -842,6 +862,31 @@ static int action_copy(DissectedImage *m, LoopDevice *d) {
                 (void) copy_times(source_fd, target_fd, 0);
 
                 /* When this is a regular file we don't copy ownership! */
+
+        } else {
+                _cleanup_strv_free_ char **list_dir = NULL;
+                size_t offset = strlen(mounted_dir);
+
```

Comment:
plese add `assert(arg_action == ACTION_LIST);` here, just to document that this is the branch where we handle the list stuff